src/custom/hooks/useApproveCallback/index.ts

```diff
@@ -30,3 +31,16 @@ export function useApproveCallbackFromTrade(
 
   return useApproveCallback(openTransactionConfirmationModal, closeModals, amountToApprove, vaultRelayer)
 }
+
+export function useApproveCallbackFromClaim(
```

Comment:
Claim functionalty's `useApproveCallback` - opted NOT to hardcode MAX GNO uint256 here and instead allow passing in values.

Reply:
fine with me, we might even need this if users start to complaint for using MAX on their investments